azure-pipelines.yml

```diff
@@ -3,19 +3,46 @@ trigger:
 - release/*
 
 jobs:
-  - job: Linux
+  - job: CMake
     timeoutInMinutes: 0
```

Comment:
Should we do a 2 hour time out (per job)?

Reply:
Yep, just did that.